shard.lock

```diff
@@ -2,5 +2,5 @@ version: 1.0
 shards:
   ameba:
     github: veelenga/ameba
-    version: 0.8.1
+    version: 0.10.0
```

Comment:
This is the crux of Crystal 0.29.0 compatibility; installing `crystal-coverage` in a Crystal application with version >= 0.29.0 gives an error as shards attempts to install `ameba#0.8.1` which fails under 0.29.0.